src/core/on-chain/index.ts

```diff
@@ -41,15 +41,16 @@ import {
 export const getOnChainTransactions = async ({
   lnd,
   incoming,
+  lookBack,
 }: {
   lnd: AuthenticatedLnd
   incoming: boolean
+  lookBack?: number
 }) => {
   try {
     const { current_block_height } = await getHeight({ lnd })
-    const after = Math.max(0, current_block_height - LOOK_BACK) // this is necessary for tests, otherwise after may be negative
+    const after = Math.max(0, current_block_height - (lookBack || LOOK_BACK)) // this is necessary for tests, otherwise after may be negative
```

Comment:
maybe we need to change LOOK_BACK from 2016 to fewer blocks in getBalance

Reply:
could be ~2 days worth of block? cron job run daily. but in case the cron job doesn't run for some reason if we cover 2 days that should work.

Reply:
note: we may not need to do any onchain check within getBalance anymore now? this would help improve the performance of the initial query on the mobile app

Reply:
ok, moved the default LOOK_BACK for 2 days and add LOOK_BACK_OUTGOING (2 blocks) and LOOK_BACK_CHANNEL_UPDATE (previous value)

Reply:
> note: we may not need to do any onchain check within getBalance anymore now? this would help improve the performance of the initial query on the mobile app

done